default/scripting/ship_designs/SD_GRAVITATING2.focs.txt

```diff
@@ -11,7 +11,7 @@ ShipDesign
         "SR_WEAPON_0_1"
         "FU_SINGULARITY_ENGINE_CORE"
         "AR_XENTRONIUM_PLATE"
-        "FU_N_DIMENSIONAL_ENGINE_MATRIX"
+        "FU_DEUTERIUM_TANK"
```

Comment:
Not sure about Dueterium, currently fuel is barely used and last we asked no one researches the techs, supply is on my long term list to balance but perhaps just extra fuel tank?